Bottom: bb0ead6c8d5a049996ec8bff4177d0f778577864
Top:    c50899f57171b8e99a6c9eec28bfb6bbae89565c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 11:29:07 +0000

Refresh of allow-polygons-with-clearlinef

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index f8c5fc6..2d858be 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -264,7 +264,7 @@ common_gui_draw_pcb_polygon (hidGC gc, PolygonType *polygon, const BoxType *clip
 
   if (TEST_FLAG (THINDRAWFLAG,     PCB) ||
       TEST_FLAG (THINDRAWPOLYFLAG, PCB) ||
-      TEST_FLAG (CLEARLINEFLAG,    PCB) ||)
+      TEST_FLAG (CLEARLINEFLAG,    PCB))
     gui->graphics->thindraw_pcb_polygon (gc, polygon, clip_box);
   else
     gui->graphics->fill_pcb_polygon (gc, polygon, clip_box);
